src/gmail.js

```diff
@@ -1255,7 +1275,7 @@ var Gmail = function(localJQuery) {
                 }
 
                 // if any matching after events, bind onreadystatechange callback
-                if(api.observe.bound(events,"after")) {
+                if(api.observe.bound(events, "after")) {
```

Comment:
This change is strictly speaking not needed, but I guess we can nitpick formatting-issues (if any) once everything else is in place 😄